client/package.json

```diff
@@ -14,7 +14,7 @@
     "pre-commit-lint": "yarn compile && yarn lint-staged",
     "test": "jest --config ./jest.config.js --maxWorkers=50% --env=jsdom",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook -s ./packages/host/public ",
+    "build-storybook": "build-storybook",
```

Comment:
offroad: popped this change in - I noticed that the demo server was unable to build storybook.

```
(node:13537) DeprecationWarning: --static-dir CLI flag is deprecated, see:

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated---static-dir-cli-flag
(Use `node --trace-deprecation ...` to show where the warning was created)
info => Loading presets
ERR! Error: Conflict when trying to read staticDirs:
ERR! * Storybook's configuration option: 'staticDirs'
ERR! * Storybook's CLI flag: '--staticDir' or '-s'
ERR!
ERR! Choose one of them, but not both.
ERR!     at buildStaticStandalone (/Users/m2/GitHub/open-msupply/client/node_modules/@storybook/core-server/dist/cjs/build-static.js:114:11)
ERR!     at async buildStatic (/Users/m2/GitHub/open-msupply/client/node_modules/@storybook/core-server/dist/cjs/build-static.js:254:5)
ERR!  Error: Conflict when trying to read staticDirs:
ERR! * Storybook's configuration option: 'staticDirs'
ERR! * Storybook's CLI flag: '--staticDir' or '-s'
ERR!
ERR! Choose one of them, but not both.
ERR!     at buildStaticStandalone (/Users/m2/GitHub/open-msupply/client/node_modules/@storybook/core-server/dist/cjs/build-static.js:114:11)
ERR!     at async buildStatic (/Users/m2/GitHub/open-msupply/client/node_modules/@storybook/core-server/dist/cjs/build-static.js:254:5)
info => Loading presets
error Command failed with exit code 1.
```